trace_decoder/tests/simulate-execution.rs

```diff
@@ -0,0 +1,34 @@
+//! Check that the [`evm_arithmetization::GenerationInputs`] produced by
+//! [`trace_decoder`] are consistent between each other, and with the block
+//! header obtained over RPC.
+
+mod common;
+
+use anyhow::Context as _;
+use common::{cases, Case};
+use libtest_mimic::{Arguments, Trial};
+use plonky2::field::goldilocks_field::GoldilocksField;
+
+fn main() -> anyhow::Result<()> {
+    let mut trials = vec![];
+    for Case {
+        name,
+        header: _,
+        trace,
+        other,
+    } in cases()?
+    {
+        let gis = trace_decoder::entrypoint(trace, other, 3, false)
```

Comment:
nit: i'm not a big fan of var names that don't carry much meaning (as opposed to for instance `inputs` / `gen_inputs` / whatever one can come up with), but I guess this isn't that important here as this is only in testing.

Reply:
Changed :)